hlib/python/hlib/README.md

```diff
@@ -0,0 +1,46 @@
+To properly use this framework, perform the following setup:
+# Using Keras
+Keras uses two different methodologies to build up a neural network: Sequential and Model. Sequential only requires users to insert the different neural layers back to back, while Model requires users to specify exactly how each layer is connected. The differences are shown below:
+```python   
+    #sequential Model
+    seq_model = Sequential()
+    seq_model.add(Dense(32, input_dim=784))
+    
+    #Normal Model
+    a = Input(shape=(32,))
+    b = Dense(32)(a)
+    mod_model = Model(inputs=a, outputs=b)
+    
+    #Save the model
+    seq_model.save("my_seq_model.h5")
+    mod_model.save("my_model.h5")
+    
+    #Load the model
+    model = load_model("my_model.h5")
+    s_model = load_model("my_seq_model.h5")
+
+    func, params = relay_parser.get_relay_model(
+            model, (32,), frontend="keras")
+    in_x = hcl.asarray(np.random.randint(1, high=10, shape=(32,)))
+    out_x = hcl.asarray(np.zeros((32,)))
+    func(in_x,*params,out_x)
+    print(out_x.asnumpy())
+```
+
+To insert a model into the HeteroCL framework, you can use *model* directly. If you want to download the model or reload it, perform the code shown in above in the bottom two lines.
+# Using HeteroCL
+1. Download and setup HeteroCL and TVM from github
+3. Once both python environments from the githubs are set up, go to the HeteroCL github, and from the main directory, go to the "python/" and "hlib/python/" folders and execute the function "python setup.py install --user" in each.
+Now that the environment is properly set up, here is how to compile a Keras model into a HeteroCL model.
+
+1. In a Python script, put into the header "from heterocl.frontend import get_relay_model".
+2. The function requires the following inputs: (*model*, *shape*, *frontend*, *dtype*, *in_params*.), where *model* is the Keras model, *shape* is the dictionary of inputs, *frontend* is the frontend being used, *dtype* is the data type, and *in_params* is an optional input if the parameters are not included in the model. The function can handle models from two different sources:
+    1. If the model was saved and exported from Keras in an HDF5 file, set ```model``` to the file path to the model.
+    2. if the model is created in the Python script, just set "model" to the Keras model output.
```

Comment:
The word should be capitalized at the beginning of a sentence. Please double-check this throughout the whole file.